docs/content/latest/explore/security/security.md

```diff
@@ -0,0 +1,122 @@
+* LDAP - use external LDAP services to perform client authentication
+* Host-based - authenticate local and remote clients based on IP address and using TLS certificates
+* Trust - authorize specific local connections
+
+The method used to authenticate a particular client connection can be selected on the basis of (client) host address, database, and user.
+
+The authentication credentials in YugabyteDB are stored internally in the YB-Master system tables. The authentication mechanisms available to clients depends on what is supported and exposed by the YSQL, YCQL, and YEDIS APIs.
+
+Read more about [how to enable authentication in YugabyteDB](../../secure/authentication).
+
+## Authorization
+
+YugabyteDB provides role-based access control, consisting of a collection of privileges on resources given to roles. 
+
+Read more about [authorization in YugabyteDB](../../secure/authorization).
+
+### Roles
+
+Roles are essential to implementing and administering access control on a YugabyteDB cluster. Roles can represent individual users or a group of users, and encapsulate a set of privileges that can be assigned to other roles (or users). Roles can be modified to grant users or applications only the essential privileges based on the operations they need to perform against the database. Typically, an administrator role is created first. The administrator then creates additional roles for users.  
+
+### Privileges
+
+Roles are explicitly granted privileges to access objects in the database using the `GRANT` command. You can, for example, assign read access to one role, data modify access to another role, and alter table access to a third.
+
+By default, when an object is created, only the owner has privileges, and other roles must be granted privileges.
+
+### Row-level access
+
+In addition to database access permissions available through the `ROLE` and `GRANT` privilege system, YugabyteDB provides a more granular level of security where tables can have row security policies that restrict the rows that users can access.
+
+Row-level Security (RLS) restricts rows that can be returned by normal queries or inserted, updated, or deleted by DML commands. Row-level security policies can be created specific to a `DML` command or with `ALL` commands. They can also be used to create policies on a particular role or multiple roles.
+
+By default, tables do not have any row level policies defined, so that if a user has access privileges to a table, all rows within the table are available to query and update.
+
+### Column-level access
+
+Column-level security in YugabyteDB is used to restrict users to viewing only a particular column or set of columns in a table. You do this by creating a view that includes only the columns that the user needs access to using the `CREATE VIEW` command.
+
+## Encryption
+
+YugabyteDB supports encryption both of clusters, and all network communication between servers. 
+
+### Encryption in transit
```

Comment:
Make sure these sub-sections are in the same order as your introductory paragraph under the parent heading...

Reply:
fixed